ui/src/views/Settings.vue

```diff
@@ -19,16 +19,27 @@
       <div class="bx--col-lg-16">
         <cv-tile :light="true">
           <cv-form @submit.prevent="configureModule">
-            <cv-text-input
+            <NsTextInput
               :label="$t('settings.phpmyadmin_path')"
-              placeholder="/phpmyadmin"
+              placeholder="e.g. /phpmyadmin"
```

Comment:
Let's use an i18n string with a parameter (for "/phpmyadmin"); "e.g." needs to be translated